pkg/services/live/live.go

```diff
@@ -208,12 +204,22 @@ func (g *GrafanaLive) Init() error {
 					cb(centrifuge.PublishReply{}, centrifuge.ErrorPermissionDenied)
 					return
 				}
-				cb(centrifuge.PublishReply{
+				centrifugeReply := centrifuge.PublishReply{
 					Options: centrifuge.PublishOptions{
-						HistorySize: reply.HistorySize,
-						HistoryTTL:  reply.HistoryTTL,
+						HistorySize: reply.StreamSize,
+						HistoryTTL:  reply.StreamTTL,
```

Comment:
since they are passed directly to HistorySize/TTL, it may be more clear to keep the names the same

Reply:
reverted it back